.github/workflows/packaging.yml

```diff
@@ -85,6 +85,11 @@ jobs:
             py_cmd: python3
           - os: ubuntu
             py_cmd: python
+        # actions/setup-python do not support MacOS runners based on ARM for Python < 3.11
+        # https://github.com/actions/setup-python/issues/906
+        exclude:
```

Comment:
Wondering if we should remove the line `architecture: 'x64'` below in the `actions/setup-python`, and bump to `actions/setup-python@v5`?

Maybe we could then remove the extra `source ./venv/bin/activate`?